### PR TITLE
Add Support for RTL languages and update the Demo project with Hebrew…

### PR DESCRIPTION
**Pull request by GuyKahlon**
## Pull request checklist

- [:heavy_check_mark:] All tests pass. Demo project builds and runs.
- [:heavy_check_mark:] I have resolved any merge conflicts.
- [:heavy_check_mark:] I have followed the [coding style](https://github.com/jessesquires/HowToContribute#style-guidelines), and reviewed the [contributing guidelines](https://github.com/jessesquires/JSQMessagesViewController/blob/develop/.github/CONTRIBUTING.md). Confirmation: ____

## What's in this pull request?

The main change is pretty simple, in JSQMessagesBubbleImageFactory.m file I add a Boolean property isRightToLeftLanguage.
I flipped for Incoming bubble depend on the language direction.

- (JSQMessagesBubbleImage *)outgoingMessagesBubbleImageWithColor:(UIColor *)color
{
 	return [self jsq_messagesBubbleImageWithColor:color flippedForIncoming:NO ^ self.isRightToLeftLanguage];
}

- (JSQMessagesBubbleImage *)incomingMessagesBubbleImageWithColor:(UIColor *)color
{
	 return [self jsq_messagesBubbleImageWithColor:color flippedForIncoming:YES ^ self.isRightToLeftLanguage];
}

Update the TextView.textAlignment property to natural in JSQMessagesCollectionViewCellIncoming and JSQMessagesCollectionViewCellOutgoing xib files.

Update Unit Tests after RTL support (The tests were failed only in device with RTL language.).

### CODE REVIEW

**Comment by GuyKahlon** [comment]
@jessesquires, hi, I realized what I did wrong, I updated my fork by merging instead of rebasing, that's why the original PR's branch had unrelated commits. I created a new branch and as you can see there is only one commit with only my changes. I would appreciate it if you could accept this PR.

Thanks, Guy.

**Comment by jessesquires** [comment]
Thanks @GuyKahlon ! Looks good! Will review soon :smile: 

Can you close the other PR and note that this one replaces it?

**Comment by GuyKahlon** [comment]
@jessesquires Done 😎

**Comment by jessesquires** [comment]
Hey @GuyKahlon ! Sorry for the delay here. If you'd like to support this PR and resolve conflicts, I'd be happy to merge! 😄

**Comment by GuyKahlon** [comment]
@jessesquires It's ok, I've merged and resolved all the conflicts.
Please let me know everything is OK.

**Comment by jessesquires** [comment]
🎉 thanks @GuyKahlon !

File: JSQMessagesViewController/Controllers/JSQMessagesViewController.m [hunk 1]
```diff
@@ -947,6 +948,7 @@ - (void)jsq_setToolbarBottomLayoutGuideConstant:(CGFloat)constant
 
 - (void)jsq_updateKeyboardTriggerPoint
 {
+    self.toolbarHeightConstraint.constant = self.inputToolbar.preferredDefaultHeight;
```
Comment (jessesquires) [inline JSQMessagesViewController/Controllers/JSQMessagesViewController.m]: this seems unrelated... ?

**Comment by jessesquires** [comment]
hey @GuyKahlon -- going to merge, but left 1 comment for you.

i'll probably revert this line manually as it seems to be unrelated...

**Comment by GuyKahlon** [comment]
Hi @jessesquires you are right, it seems unrelated, maybe some merge mistake.
Please ignore this line.
Thanks
